meta-balena-common/recipes-support/os-helpers/os-helpers/os-helpers-tpm2

```diff
@@ -96,17 +105,35 @@ parse_pcr7_eventtypes() {
     awk '
         /PCRIndex: 7/ {flag=1; next}
         /EventNum:/ {flag=0}
-        flag && /EventType:/ {gsub("\"", "")print $2}'
+        flag && /EventType:/ {gsub("\"", ""); print $2}'
 }
 
 firmware_measures_efibins() {
-    [ -f "${TPM_EVENTLOG_PATH}" ] || exit 2
-    tpm2_eventlog "${TPM_EVENTLOG_PATH}" \
-        | parse_pcr7_eventtypes \
-        | grep -e EV_EFI_VARIABLE_AUTHORITY
+    if [ ! -f "${TPM_EVENTLOG_PATH}" ]; then
+        echo "unknown"
+    elif tpm2_eventlog "${TPM_EVENTLOG_PATH}" \
+            | parse_pcr7_eventtypes \
+            | grep -e EV_EFI_VARIABLE_AUTHORITY \
+            >/dev/null; then
+        echo "measured"
+    else
+        echo "unmeasured"
+    fi
 }
 
-
+print_pcr_val_bin() {
+    PCRS=$1
+    PCR_VAL_BIN=$2
+    DIGEST_SIZE=32
+    SKIP=0
+    printf " %s:\n" "sha256"
```

Comment:
Nitpick: Is there a reason to use `printf` here and not `echo` like everywhere else? `printf` can be useful for e.g. formatting hex numbers, but here it just seems to add strings together.